File: 21 - Determine the Longest Sequence of Consecutive Ones in a Binary Array/solution.py
def find_longest_sequence(nums):
    # Your code here
    max = 0
    count = 0

    for num in nums:
        if num == 1:
            count += 1
            if count > max:
                max = count
        else:
            count = 0

    return max


#   Test cases
print(find_longest_sequence([1, 1, 0, 1, 1, 1]))

File: 21 - Determine the Longest Sequence of Consecutive Ones in a Binary Array/solution.js
function findLongestSequence(nums) {
  // Your code here
  let max = 0;
  let count = 0;

  for (let num of nums) {
    if (num === 1) {
      count++;
      if (count > max) max = count;
    } else {
      count = 0;
    }
  }

  return max;
}

//   Test cases
console.log(findLongestSequence([1, 1, 0, 1, 1, 1]));
